blockchain/evm.go

```diff
@@ -103,47 +103,52 @@ func StringToBytes32(jobid string) string {
 }
 
 func logEventToOracleRequest(log eth.Log) (models.JSON, error) {
-	data := log.Data
+	cborData, dataPrefixBytes, err := logDataParse(log.Data)
+	if err != nil {
+		return models.JSON{}, err
+	}
+	js, err := models.ParseCBOR(cborData)
+	if err != nil {
+		return js, fmt.Errorf("error parsing CBOR: %v", err)
+	}
+	return js.MultiAdd(models.KV{
+		"address":          log.Address.String(),
+		"dataPrefix":       bytesToHex(dataPrefixBytes),
+		"functionSelector": models.OracleFulfillmentFunctionID20190128withoutCast,
+	})
+}
+
+func logDataParse(data eth.UntrustedBytes) (cborData []byte, dataPrefixBytes []byte, rerr error) {
```

Comment:
It looks like named return variables are used here, but not really utilized. Suggest fully utilizing this or just declare variables inside the function.

Reply:
you r right. this is just to make the function declaration more clear when a function is returning multiple results with same type. I found it pretty helpful myself